### DISCUSSION TITLE
HLS streams from rtsp source not loading in Emby.

### DISCUSSION THREAD

**Discussion by Wheemer** [bluenviron/mediamtx#882] [Questions]
Hello;

Great software, really love it.

I have one rtmp source published to simple from obs and it loads hls perfectly in emby and works very well. However I have three rtsp sources that will never load in emby via hls. If it something to do with the audio channels? I notice in obs I cannot turn off the audio, but on the camera rtsp feeds I have audio disabled. I did try enabling audio but it still reported 0 Channels in Emby, while the rtmp source reports 1 channel mono.

Thanks for any help.

**Comment by Wheemer** [comment]
This seems to be working:

entrance:
    runOnInit: ffmpeg -i rtsp://192.168.1.140/live/ch00_1 -f lavfi -i sine=frequency=0:sample_rate=48000 -c:v copy -f rtsp rtsp://127.0.0.1:$RTSP_PORT/$RTSP_PATH

Does it look right to you?

**Reply by aler9** [reply]
hello, this is a bug that has already been reported here #816 - there are some clients that refuses to play HLS unless an audio track is present. When i'll find a way to replicate the issue, i'll also be able to fix it.

**Reply by Wheemer** [reply]
OK thanks, but in the meantime I am working well with the command I typed above.

**Comment by Wheemer** [comment]
Just an update, I frequently have cameras drop with the above method. Going back to regular rtsp links for now, hopefully there's a new version soon, for the issue you mention is the same.

**Reply by aler9** [reply]
there's a nightly release here:
https://github.com/aler9/rtsp-simple-server/issues/816#issuecomment-1073278102

**Reply by Wheemer** [reply]
Working very well now, thanks so much. This is a very nice program, good work.

**Comment by Wheemer** [comment]
Still seeing issues with this. With the ffmpeg source the streams started instantly from all clients I tested. However now I frequently see playback errors or the streams take a long time to start. I am seeing the ts being created multiple times while watching the stdout. Also when the stream starts it sometimes ends with a playback error.

**Reply by Wheemer** [reply]
I just see it serving the index and stream over and over and maybe eventually it plays.

**Reply by Wheemer** [reply]
I have also inquired with the emby team, it could be something with a recent update.

**Reply by aler9** [reply]
i tried debugging with Emby, unfortunately it requires a paid license.

**Reply by Wheemer** [reply]
https://emby.media/community/index.php?/topic/107125-problem-with-hls/#comment-1127469

We are going through it now. Strange that debugging would need a license.

**Reply by Wheemer** [reply]
The issue doesn't seem to be gaining traction currently over on the emby forum. If I provided you with an admin login to my emby would that help or be beneficial to you?

**Reply by Wheemer** [reply]
It seems my simple server it not working for hls any longer at all. I can't even load in vlc anymore. And rtsp seems to be taking longer to start but eventually works.

[a comment by an automated account is not shown]
